src/test/java/org/springframework/data/couchbase/core/CouchbaseTemplateQueryIntegrationTests.java

```diff
@@ -124,7 +126,7 @@ void findByMatchingQuery() {
 
 		couchbaseTemplate.upsertById(User.class).all(Arrays.asList(user1, user2, specialUser));
 
-		Query specialUsers = new Query(QueryCriteria.where("firstname").like("special"));
+		Query specialUsers = new Query(QueryCriteria.where(i("firstname")).like("special"));
```

Comment:
It might already be in the PR, but I wasn't paying close enough attention

But can the value methods take `N1QLExpression`'s to? I can imagine that somewhere someone might want to use a built in N1QL method to compute the value to compare against..